routes/cube_routes.js

```diff
@@ -22,14 +22,19 @@ var {
   build_tag_colors,
   maybeCards,
   getElo,
+  CSVtoCards,
+  generateBlogpost,
+  populateCardDetails,
+  compareCubes,
 } = require('../serverjs/cubefn.js');
 const analytics = require('../serverjs/analytics.js');
 const draftutil = require('../dist/utils/draftutil.js');
 const cardutil = require('../dist/utils/Card.js');
 const carddb = require('../serverjs/cards.js');
 carddb.initializeCardDb();
 const util = require('../serverjs/util.js');
-const { GetPrices } = require('../serverjs/prices.js');
+const prices = require('../serverjs/prices.js');
+const { GetPrices } = prices;
```

Comment:
this is not a good pattern. Is this from mixing default and named exports? Can we not do this?